iceoryx_posh/test/moduletests/test_roudi_service_registry.cpp

```diff
@@ -91,198 +186,199 @@ TEST_F(ServiceRegistry_test, AddMoreThanMaximumNumberOfServiceDescriptionsFails)
 
     for (auto& service : services)
     {
-        auto result = sut.add(service);
+        auto result = this->sut.add(service);
         ASSERT_FALSE(result.has_error());
     }
 
-    auto result = sut.add(iox::capro::ServiceDescription("Foo", "Bar", "Baz"));
+    auto result = this->sut.add(iox::capro::ServiceDescription("Foo", "Bar", "Baz"));
     ASSERT_TRUE(result.has_error());
     EXPECT_THAT(result.get_error(), Eq(ServiceRegistry::Error::SERVICE_REGISTRY_FULL));
 }
 
-TEST_F(ServiceRegistry_test, AddServiceDescriptionsWhichWasAlreadyAddedAndReturnsOneResult)
+TYPED_TEST(ServiceRegistry_test, AddServiceDescriptionsWhichWasAlreadyAddedAndReturnsOneResult)
 {
-    ::testing::Test::RecordProperty("TEST_ID", "d8f61eb2-d082-4c26-9970-461427c3d200");
-    auto result1 = sut.add(ServiceDescription("Li", "La", "Launebaer"));
+    ::testing::Test::RecordProperty("TEST_ID", "a911f654-8314-4ea3-b9b2-1afa121a2b21");
```

Comment:
You copied the test id from the test above.

Reply:
We need some kind of automation for this. It is impossible to the reviewer to catch such stuff. A new test could be based on any test from the file and one would need to manually check if the id is unique. What makes things even worse, one could even copy a test from a different file

Reply:
Agreed.